warehouse/query-core/src/main/java/datawave/query/language/functions/jexl/Compare.java

```diff
@@ -0,0 +1,60 @@
+package datawave.query.language.functions.jexl;
+
+import com.google.common.collect.Sets;
+import datawave.query.language.functions.QueryFunction;
+
+import java.util.Set;
+
+/**
+ * The #COMPARE function is used to compare the values of two fields.
+ * <p>
+ * Function args are a field name, an operator, a mode, and another field name.
+ * <p>
+ * Lucene Example: #COMPARE(FIELD_A, &lt;, ANY, FIELD_B)
+ * <p>
+ * Jexl Example: filter:compare(FIELD_A, &lt;, ANY, FIELD_B)
+ */
+public class Compare extends AbstractEvaluationPhaseFunction {
+    
+    private final Set<String> validOpArgs = Sets.newHashSet("<", "<=", ">", ">=", "==", "=", "!=");
+    private final Set<String> validModeArgs = Sets.newHashSet("ANY", "ALL");
+    
+    public Compare() {
+        super("compare");
```

Comment:
How about a constant for "compare".  I see it used below as well.

Reply:
As-is this follows the pattern of the include/exclude functions. I can extract that to a constant if you prefer

Reply:
its fine as is